imessage/bluebubbles/api.go

```diff
@@ -1568,7 +1568,7 @@ func (bb *blueBubbles) downloadAttachment(guid string) (attachment *imessage.Att
 	}
 
 	return &imessage.Attachment{
-		GUID:       guid,
+		GUID:       guid, // had trouble renaming this one (guid)
```

Comment:
and do we want to rename stuff like this when it's going into a struct with the same capitalization as its key? Felt a bit strange to do that. (will remove the added comment)

Reply:
Did you update the function's input parameter on line 1538?

Reply:
No I did not. The question was whether or not we want to rename this variable in all the places that it appears.

I ask because I made the change and the result looks a bit strange so I thought I'd ask in case this might constitute an exception. The result for _that_ line of course being `GUID: GUID,`.

Totally happy to make the change, just looking for guidance. I also left functions/variables unchanged if capitalizing changed them to exported ones.

Reply:
Yeah, this has a huge renaming effect throughout the code and I don't think we need to do all of them right now. You changed a bunch and that's fine. Leave this as is.